Production/test/lnbatch.sh

```diff
@@ -25,3 +25,8 @@ fi
 
 mkdir -p $TODIR
 lndir -silent -ignorelinks $FROMDIR $TODIR	
+
+if [[ "${TODIR}/CACHEDIR.TAG" ]]; then
```

Comment:
shouldn't this be an actual test like like -L

Reply:
Darn! You're right. I forgot to go back and add that. It's dded now.